cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -634,13 +634,15 @@ recompiled and re-deployed.)
    - Using Yum (on one line)::
 
        $ sudo yum install cdap cdap-gateway \
-             cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-1.0 cdap-hbase-compat-1.0-cdh \
              cdap-kafka cdap-master cdap-security cdap-ui
 
    - Using APT (on one line)::
 
        $ sudo apt-get install cdap cdap-gateway \
-             cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-1.0 cdap-hbase-compat-1.0-cdh \
```

Comment:
Yes, same as above.  Our approach so far has been to keep the docs simple and install all packages, then let the startup scripts figure out the right dependency.